### PR TITLE
Transmitter usage fix

### PR DESCRIPTION
**Pull request by zhvkgj**

### CODE REVIEW

File: lzy/storage-api/src/main/java/ai/lzy/storage/StorageClientFactory.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                // blank
+            }
+        };
+
+        stop.accept(transferPool);
+        stop.accept(chunkPool);
+        stop.accept(consumePool);
+    }
+
+    public interface Provider<T extends StorageClient> {
```
Comment (imakunin) [inline lzy/storage-api/src/main/java/ai/lzy/storage/StorageClientFactory.java:70]: why not `Supplier<T>`?

File: test/src/test/java/ai/lzy/test/scenarios/PyApiTest.java [hunk 2]
```diff
@@ -22,6 +22,11 @@ public class PyApiTest {
         CondaEnvironment.reconfigureConda(false);  // To optimize conda configuration
     }
 
+    @Test
+    public void testOleg() {
```
Comment (imakunin) [inline test/src/test/java/ai/lzy/test/scenarios/PyApiTest.java:26]: it's fanny, but absolutely unclear what this is about :)

Comment (zhvkgj) [reply test/src/test/java/ai/lzy/test/scenarios/PyApiTest.java:26]: it should be removed:)

File: lzy/storage-api/src/main/java/ai/lzy/storage/StorageClientFactory.java (hunk 1 above)
Comment (zhvkgj) [reply lzy/storage-api/src/main/java/ai/lzy/storage/StorageClientFactory.java:70]: it doesn't matter, let it be supplier

**Pull request merged**